app/templates/components/public/ticket-list.hbs

```diff
@@ -121,13 +121,20 @@
                     <td>{{currency-symbol this.eventCurrency}} {{this.orderAmount.tax.amount}}</td>
                   </tr>
                 {{/if}}
-                <tr><td>{{t 'Total'}}</td><td>{{currency-symbol this.eventCurrency}} {{format-money this.orderAmount.total}}</td></tr>
+                <tr><td>{{t 'Total'}}</td>
+                  {{#if this.user}}
```

Comment:
This should only be shown in the case user is the event organizer

Reply:
Yes this is shown in case user is event organizer only.
```javascript
@user={{true}}
```
This is passed in Ticket List components in add-order.hbs file so only events organized by user have this editable feature.

Reply:
Then please change the prop name. `user=true` does not make sense. Should be `amountEditable=true`

Reply:
I have changed it.
Please check it and please let me know to make any changes more.